client/landing/jetpack-cloud/sections/scan/main.tsx

```diff
@@ -189,7 +190,7 @@ class ScanPage extends Component< Props > {
 				{ this.renderHeader( translate( 'Something went wrong' ) ) }
 				<p>
 					{ translate(
-						'The scan was unable to process the themes directory and did not complete ' +
+						'The scan did not complete ' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **2** times:
`translate( 'The scan was unable to process the themes directory and did not complete successfully. In order to complete the scan you will need to speak to support who can help determine what went wrong.' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/jetpack-scanner-last-scan-time)